src/app/features/task-repeat-cfg/task-repeat-cfg.service.ts

```diff
@@ -271,4 +319,23 @@ export class TaskRepeatCfgService {
       isAddToBottom: taskRepeatCfg.order > 0,
     };
   }
+
+  private _getSubTaskRepeatTemplate(
+    taskRepeatCfg: TaskRepeatCfg,
+    aParentId: string,
+  ): Task {
+    const isAddToTodayAsFallback =
+      !taskRepeatCfg.projectId && !taskRepeatCfg.tagIds.length;
+    return this._taskService.createNewTaskWithDefaults({
+      title: taskRepeatCfg.title,
+      additional: {
+        repeatCfgId: taskRepeatCfg.id,
```

Comment:
I would just leave out the repeatCfgId part

Reply:
Please see comment below